samples/graph-chat-lifecycle/nodejs/config/default.json

```diff
@@ -0,0 +1,7 @@
+{
+    "tab": {
+        "appId": "YOUR-MICROSOFT-APP-ID",
+        "appPassword": "YOUR-MICROSOFT-APP-PASSWORD"
+    },
+    "port": "3333"
```

Comment:
is this port required? i see app running over port number `3978`

Reply:
Changed that to 3333 port number everywhere.

Reply:
I could still see 3978 in Read Me instructions. Could you please use 3978 as we are using same consistently across samples.